diesel/src/associations/belongs_to.rs

```diff
@@ -14,7 +14,66 @@ pub trait BelongsTo<Parent> {
     fn foreign_key(&self) -> Option<&Self::ForeignKey>;
     fn foreign_key_column() -> Self::ForeignKeyColumn;
 }
-
+/// `grouped_by` is called on a `Vec<Child>` with a `&Vec<Parent>` and returns a `Vec<Vec<Child>>`
```

Comment:
app empty line above this doc comment

Reply:
First doc comment paragraph is the summary line and should not be longer than one small sentence